### DISCUSSION TITLE
docker and docker-compose build packs unavailable

### DISCUSSION THREAD

**Discussion by sorokinvld** [coollabsio/coolify#1127] [Q&A] [answered]
Hi! I have docker and docker-compose options unavailable in my project. How it can be fixed? 
Thank you in advance

**Comment by davay42** [comment]
Same thing, can't figure out how to make them available for a build.

**Reply by andrasbacsai** [reply]
Is this related to v3 or v4?

**Reply by davay42** [reply, marked as answer]
Yeah, it was v4 related. I believe you've made these build packs available there since then. Yet, I didn't have time to wait for that or study nixpacks from scratch. So I've decided to go with v3 for now and the experience there is still very nice! Looking forward a full v4 release with proper v3 migration guide to transition later on.
